client/src/app/core/repositories/agenda/speaker-repository.service.ts

```diff
@@ -21,7 +24,23 @@ export class SpeakerRepositoryService extends BaseRepositoryWithActiveMeeting<Vi
     }
 
     public getFieldsets(): Fieldsets<Speaker> {
-        return { [DEFAULT_FIELDSET]: ['begin_time', 'end_time', 'weight', 'note', 'speech_state', 'point_of_order'] };
+        const statisticsFieldset: (keyof Speaker)[] = [
+            'begin_time',
+            'end_time',
+            'point_of_order',
+            'speech_state',
+            'user_id'
+        ];
+        const defaultSet: (keyof Speaker)[] = [
+            'begin_time',
+            'end_time',
+            'weight',
+            'note',
+            'speech_state',
+            'point_of_order'
+        ];
```

Comment:
Both fieldsets share same keys of `Speaker`. You can make a fieldset, that contains only the four keys, that are required and "extend" this fieldset by the keys, which are used for `statisticsFieldset` and `defaultSet`.